### DISCUSSION TITLE
Allow for EGL Texture Sharing (like in OpenTK branch)

### DISCUSSION THREAD

**Discussion by llfab** [AvaloniaUI/Avalonia#8556] [Q&A] [answered]
Like mentioned in [Integration of Vulkan Renderer](https://github.com/AvaloniaUI/Avalonia/discussions/7137) we are working on our Vulkan renderer integration into UI frameworks. We had made that work for both WGL (using official Avalonia package) and EGL (using the Avalonia opentk-integration branch. 

The latter was necessary because only that branch would allow for the right context sharing with EGL. As WGL does not work so well on Windows, EGL would be our favorite. The open-tk branch seems to not have been touched for a while. Hence, I wonder whether at least the great improvements regarding EglContext, GlInterface, etc. could be integrated in to the master for a next release. 

@kekekeks : what says you?

**Comment by kekekeks** [comment, marked as answer]
Planned for 11.0

**Reply by llfab** [reply]
Nice. Any (rough) timeline on this like 2022, 2023, ...?

**Reply by kekekeks** [reply]
This fall

**Reply by llfab** [reply]
Great. So the plan would be to integrate what is in the opentk-integration branch or just something that is similar?
